htdocs/comm/propal/card.php

```diff
@@ -1267,7 +1267,7 @@
 					$mesg = $langs->trans("CantBeLessThanMinPrice", price(price2num($price_min, 'MU'), 0, $langs, 0, 0, -1, $conf->currency));
 					setEventMessages($mesg, null, 'errors');
 					$error++;
-				} elseif ($pu_equivalent_ttc && $price_min_ttc && ((price2num($pu_equivalent_ttc) * (1 - $remise_percent / 100)) < price2num($price_min_ttc))) {
+				} elseif ($tva_tx == $prod->tva_tx && $pu_equivalent_ttc && $price_min_ttc && ((price2num($pu_equivalent_ttc) * (1 - $remise_percent / 100)) < price2num($price_min_ttc))) {
```

Comment:
This means that if product that has a default vat of 20, when sold to a foreign country (where vat is 0), the test on minimum prices will be completely disabled ?

Reply:
The test on minimal price *including vat* is averted, but the test on the price *excluding vat* remains untouched

Whatever is decided here, it might be a good idea to also do it on commande and facture cards.

Reply:
Hey there @eldy, what do you think we should do here. I see several options:

- we can reapply the vat rate specific to this proposal to check the minimum price. Which would mean:
  - comparing `$tva_tx` to `$prod->tva_tx`
  - if `$tva_tx` is inferior, make `$pu_equivalent_ttc` the vat included price recalculated as if it had the `$prod->tva_tx` rate for VAT
- ignore `$pu_equivalent_ttc` altogether (I must admit that I fail to see why we compare both vat excluded and included price)
- use the solution in this PR
- the answer D

Thanks for your input here

Reply:
If the price was set in HT, it means the reference price is the price without tax, son we should check the minimum sell price without tax only.
If the price was set including tax, then we should check the price with tax.